src/main/java/net/mcreator/element/types/Dimension.java

```diff
@@ -78,6 +81,10 @@ public Dimension(ModElement element) {
 		this.sleepResult = "ALLOW";
 	}
 
+	public boolean customSky() {
+		return !skyTexture.isEmpty();

```

Comment:
Probably should return true if any of the sun/moon/sky textures is not empty

Reply:
I know but I made so when you only enter 1 or 2 textures (and not all 3) you cannot confirm the Dimension.

Reply:
Still, this check should include all cases, as idea is to decouple frontend and backend logic and be self-sufficient.

Reply:
Actually, if you require all to be present, check that all of them are not empty.